### PR TITLE
Enable MCS Testing

### PR DESCRIPTION
**Pull request by lrennels**
@jrising I'm pulling out the mcs testing here, without actually using the updated Mimi and thus without updating the mcs code.  This way we can feel confident that the tests pass _before_ the Mimi changes, and then the same ones pass _after_ the Mimi change!

So the plan would be first merge this one (#77), then that one (#74)

### CODE REVIEW

File: test/test_scc.jl [hunk 1]
```diff
@@ -10,23 +9,29 @@ uds = [undiscount_scc(m, 2020), undiscount_scc(m, 2040), undiscount_scc(m, 2075)
...
-    sccsd = std(sccs) / sqrt(1000)
+    sccsd = std(sccs) / sqrt(100_000)
     @test all(sccerror .< 3*sccsd) # average off by < 3x SD
 end
 
 m = getpage("RCP4.5 & SSP2", true, true, true)
-sccs = compute_scc_mcs(m, 1000, year=2020)
-checksccs(sccs, [37.031, 89.742, 161.868, 299.439, 847.020, 261.739])
+sccs = compute_scc_mcs(m, 100_000, year=2020)
+# TODO - get the baseline values to test against!
```
Comment (lrennels) [inline test/test_scc.jl:21]: @jrising I don't think this file has worked in a while, given there was a bug in the syntax and it probably was holding old PAGE2009 values but I'm really not sure?  As of now with a n of 1000 the tests weren't passing even on master so I'm thinking I should re-run with larger n and remake these tests?  Do you use these regularly currently?

File: test/test_scc.jl [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
 end
 
+n = 1000
+
 m = getpage("RCP4.5 & SSP2", true, true, true)
-sccs = compute_scc_mcs(m, 1000, year=2020)
-checksccs(sccs, [37.031, 89.742, 161.868, 299.439, 847.020, 261.739])
+sccs = compute_scc_mcs(m, n, year=2020)
+# TODO - get the baseline values to test against!
+# checksccs(sccs, [37.031, 89.742, 161.868, 299.439, 847.020, 261.739]; n = n)
```
Comment (lrennels) [inline test/test_scc.jl:22]: @jrising I don't think this file has worked in a while, given there was a bug in the syntax and it probably was holding old PAGE2009 values but I'm really not sure? I can try to get new baseline values with a high `n` or something, but as of now they swing more than allowed with an n of 1000 ...

Comment (lrennels) [reply test/test_scc.jl:22]: alternatively we leave that for a new PR, and merge this and #74 first, since this really isn't related to those

Comment (lrennels) [reply test/test_scc.jl:22]: ^ I'd probably recommend that to keep things clean and moving along

Comment (jrising) [reply test/test_scc.jl:22]: Oh, you're probably right that it hadn't been updated. PAGE-ICE didn't publish SCC values, so there was nothing to benchmark against. I can clean this up later, but let's not worry about it for what you're doing now.

Comment (lrennels) [reply test/test_scc.jl:22]: Great, we can leave it how it is for this merge ... it'll test that the functionality runs but won't compare it to anything

**Review by jrising — APPROVED** [review]
Awesome! Thanks for pulling this out. It all looks good to me.

**Comment by lrennels** [comment]
Great! I guess I don't have write access so you'll need to merge :)

**Comment by jrising** [comment]
Merged! And I'll give you write access.

**Pull request merged**